android/src/main/java/org/devio/rn/splashscreen/SplashScreen.java

```diff
@@ -23,13 +30,89 @@
 public class SplashScreen {
     private static Dialog mSplashDialog;
     private static WeakReference<Activity> mActivity;
+    private static boolean isVideoActive = false;
+    private static boolean isImageActive = false;
+
+    public static void showVideo(final Activity activity) {
+        if (activity == null) return;
+        if (mSplashDialog != null) return;
+        if (isImageActive || isVideoActive) return;
+        isVideoActive = true;
+
+        mActivity = new WeakReference<Activity>(activity);
+        activity.runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+                if (!activity.isFinishing()) {
+                    mSplashDialog = new Dialog(activity, R.style.SplashScreen_Fullscreen);
+                    mSplashDialog.setContentView(R.layout.video_view);
```

Comment:
Can't find this xml

Reply:
Will the layout be part of Exodus app structure?

Reply:
Yes, check updated docs. I added it in refs.xml

Reply:
I think this library and project are very heavily interlinked. I think either we change the code to not rely on the app source or we should move it into the project.

Reply:
They interlinked, but no more than before. From the main project we take:
* video_view layout -> I guess we can move this to the project, don't see a downside
* video content -> it is expected this is a requirement. We could pass it in as an argument instead, but the amount of linkage would be the same (& we change existing pattern of the lib)

Reply:
Just checked. Didn’t know about it

Reply:
9904972fea067890fb896172ce2ced0496d90b27